src/xdocs/checks/javadoc/writetag.xml

```diff
@@ -201,13 +203,14 @@ public class Test { // violation as required tag is missed
 * Some class
 * @since 1.2
 */
-public class Test {
+public class Example4 {
+  // violation 2 lines below 'Type Javadoc tag @since'
   /** some doc
-  * @since violation
+  * @since
   */
-  void foo() {}
+  void testMethod1() {}
 
-  public void method() {}
+  public void testMethod2() {}
```

Comment:
Please put above this method `/** some doc */`.

And make all examples1-3 to be "same" as  example4.

Reply:
Done